bin/BuildPackagesHelper.g

```diff
@@ -0,0 +1,7 @@
+MakeReadWriteGlobal("SetPackageInfo");
```

Comment:
While we don't have a perfect place to put this file and I don't really like it in 'bin', we could put it in 'etc'..

I'd probably just put it in a place as a here document (assuming that doesn't lead to some other problem), to avoid having to decide where to put it.

Reply:
> I'd probably just put it in a place as a here document (assuming that doesn't lead to some other problem), to avoid having to decide where to put it.

I'm sorry, I don't understand what you are saying here. Can you explain more?